src/drivers/evdev/lv_evdev.c

```diff
@@ -150,6 +150,10 @@ static void _evdev_read(lv_indev_t * indev, lv_indev_data_t * data)
             data->state = dsc->state;
             data->key = dsc->key;
             break;
+        case LV_INDEV_TYPE_BUTTON:
+            data->state = dsc->state;
+            data->btn_id = (dsc->state == LV_INDEV_STATE_PRESSED) ? 0 : 1;
```

Comment:
Why does it alter between `btn_id` 0 and 1? Shouldn't it be always `btn_id = 0` and just change the `state`?